### DISCUSSION TITLE
What does the "view" button do

### DISCUSSION THREAD

**Discussion by zyegao** [cyberbotics/wrestling#87] [Q&A] [answered]
How can I see the corresponding video when I lose or win? Click the "view" button to see the video of a long time ago, not the video of the current game, and the opponents in it are not the opponents above my ranking

**Comment by MJavadZallaghi** [comment]
@omichel 
True, sometimes animations are not updated exactly after an action complete evaluation.

**Comment by omichel** [comment, marked as answer]
The "View" button allows you to watch a game in 3D on the browser. This should work with any recent web browser supporting [WebGL 2.0](https://caniuse.com/webgl2) on a desktop computer, except may for Safari. On mobile phones or tablets, it may not work if you are using Safari/iPhone/iPad or have an old phone.

@MJavadZallaghi: yes, it takes some time to upload the animations (and the old version of the animation may also be cached in your browser cache). So please wait a little bit and clear the cache of your browser to view the latest animations.
